api/cosmos/app/v1alpha1/query_grpc.pb.go

```diff
@@ -18,6 +18,10 @@ import (
 // Requires gRPC-Go v1.32.0 or later.
 const _ = grpc.SupportPackageIsVersion7
 
+const (
+	Query_Config_FullMethodName = "/cosmos.app.v1alpha1.Query/Config"
```

Comment:
The new `protoc-gen-go-grpc` `v1.3.0` adds constants instead of reusing a string of the same argument.
This is the only change thorough the generated files.